dataikuapi/dss/project.py

```diff
@@ -1035,17 +1034,34 @@ def get_api_service(self, service_id):
     ########################################################
 
     def list_exported_bundles(self):
+        """
+        :returns: A dictionary of all bundles for a project on the Design node.
+        """
         return self.client._perform_json("GET",
                 "/projects/%s/bundles/exported" % self.project_key)
 
     def export_bundle(self, bundle_id):
+        """
+        Creates a new project bundle on the Design node 
+        
+        :param str bundle_id: bundle id tag 
+        """
         return self.client._perform_json("PUT",
                 "/projects/%s/bundles/exported/%s" % (self.project_key, bundle_id))
 
     def get_exported_bundle_archive_stream(self, bundle_id):
         """
         Download a bundle archive that can be deployed in a DSS automation Node, as a binary stream.
-        Warning: this stream will monopolize the DSSClient until closed.
+        
+        .. warning::
+            
+            this stream will monopolize the DSSClient until closed, so the stream must be closed after use or used within a ``with`` statement. For example:
```

Comment:
I'd drop the "this stream will monopolize the DSSClient until closed" and just say what needs to be done

Reply:
👍  updated